functionary/prompt_template/base_template.py

```diff
@@ -35,11 +41,13 @@ def get_stop_token_for_function_parameter(
         """
         raise NotImplementedError
 
-    def get_predefined_function_names(self) -> List[str]:
+    def get_predefined_function_names(self, function_types: Any) -> List[str]:
```

Comment:
this will be conflicted with: https://github.com/MeetKai/functionary/blob/code-gen-inference/functionary/inference.py#L77C45-L77C74 ?

Reply:
Fixed